pkg/ccl/importccl/load.go

```diff
@@ -369,12 +369,10 @@ func insertStmtToKVs(
 		if err != nil {
 			return errors.Wrapf(err, "process insert %q", insertRow)
 		}
-		// TODO(bram): Is the checking of FKs here required? If not, turning them
-		// off may provide a speed boost.
 		// TODO(mgartner): Add partial index IDs to ignoreIndexes that we should
 		// not add entries to.
 		var ignoreIndexes util.FastIntSet
-		if err := ri.InsertRow(ctx, b, insertRow, ignoreIndexes, true, row.CheckFKs, false /* traceKV */); err != nil {
```

Comment:
Note - this `CheckFKs` wasn't actually doing anything because the inserter was created with `SkipFKs` above.